index.bs

```diff
@@ -4005,8 +4005,11 @@ This [=registration extension=] and [=authentication extension=] enables use of
 
     Servers supporting UVI extensions MUST support a length of up to 32 bytes for the UVI value.
 
-    Example for [=authenticator data=] containing one UVI extension
+    Example for [=authenticator data=] containing one UVI extension along with CDDL
```

Comment:
well, I spose I'd write it this way:
Examples, in [=CDDL=] and of the CBOR-encoded output, of [=authenticator data=] containing one UVI extension:

Reply:
@equalsJeffH 

>question: given the discussion on the webauthn call a couple days ago (see: #679 (comment)), do we still wish to do this?

I think we need to first determine: what the constraints are on UVI size.  The current text seems to imply 32 bytes, but as @rlin1 pointed out, that may restrict stronger hashes.

Based on whatever is decided, then we can determine whether a  CDDL description required for this extension.  If the size is unconstrained, I would say a CDDL description adds no value and the current text related to CBOR usage is sufficient ("The UVI is encoded as CBOR byte string (type 0x58)").  

If the size is constrained, then a CDDL description could add value from a canonical perspective.